### PR TITLE
Add announcements related to cgroup implementation on gizmo

### PR DESCRIPTION
**Pull request by atombaby**
An informational page on cgroups and a brief discussion of changes to grabnode

### CODE REVIEW

**Review by dtenenba — APPROVED** [review]
👍

**Review by vortexing — APPROVED** [review]
Once it's not listed as a draft PR, we can merge it!  Also.... exciting!

File: _scicompannounce/2021-08-08-gizmo-cpu-cgroup.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+> Effective: 8 August 2021
+
+In this first phase we are only going to manage CPU and GPU resources.  When your job starts, it will be placed into a cgroup which is only allowed to use the number of CPUs and GPUs requested.
+
+### What to Watch For
+
+#### Default Threads
+
+Your job may slow down if it has previously used more resources than requested.  This is often due to the default behavior of some tools and libraries- OpenBLAS is a frequent offender as it by default uses all available cores.  Picard also defaults to the all cores on a host.
```
Comment (bmcgough) [inline _scicompannounce/2021-08-08-gizmo-cpu-cgroup.md:39]: Typo - "to the all cores" remove "the"

**Review by bmcgough — APPROVED** [review]
Looks good, thank you!

**Pull request merged**